src/containers/dashboard/Dashboard.js

```diff
@@ -62,6 +67,33 @@ const ButtonWrapper = styled.div`
   display: flex;
 `;
 
+const NavButtonGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 200px);
+  grid-gap: 0;
+  margin-bottom: 20px;
+`;
+
+const NavButton = styled(Button)`
+  background-color: ${(props) => (props.selected ? PURPLE.P300 : NEUTRAL.N50)};
+  border-color: ${(props) => (props.selected ? PURPLE.P300 : NEUTRAL.N50)};
+  border-radius: 0;
+  color: ${(props) => (props.selected ? 'white' : NEUTRAL.N700)};
+
+  &:hover {
+    background-color: ${(props) => (props.selected ? PURPLE.P500 : NEUTRAL.N200)};
+    border-color: ${(props) => (props.selected ? PURPLE.P500 : NEUTRAL.N200)};
+  }
+
+  &:first-child {
+    border-radius: 3px 0 0 3px;
+  }
+
+  &:last-child {
+    border-radius: 0 3px 3px 0;
+  }
+`;
```

Comment:
Based on the attached gif, your styles look broken. Why don't you just use `ButtonGroup`?

https://material-ui.com/components/button-group

Reply:
oops. ah good point about `ButtonGroup`... i can also add to LUK if that would be useful?